forc-plugins/forc-doc/src/subdirectory_test/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "lib.sw"
+license = "Apache-2.0"
+name = "subdirectory_test"
```

Comment:
Do we run this test as a part of CI anywhere? If this is just for local testing, maybe it's worth omitting?

Reply:
Good point. I'll remove it, and maybe there's another way I can add this as a unit test. This was just the first (easiest) thing that came to mind for testing nested submodules.

Reply:
Suggestion: we can also put this under `forc-doc/test/data/` as well (or something along those lines) and we can create an issue to utilize this in forc-doc specific tests in a followup PR.